### ISSUE TITLE
Implement comment and user meta endpoints

### ISSUE THREAD

**Issue by danielbachhuber** [WP-API/WP-API#1229]
Now that we have a good pattern for post meta, we should implement comment and user meta endpoints.

Blocked by #1144

**Comment by rachelbaker** [comment]
Punted to Beta 4

**Comment by danielbachhuber** [comment]
We'll also need Term meta

**Comment by danielbachhuber** [comment]
Closing in favor of https://github.com/WP-API/wp-api-meta-endpoints/issues/1
